### DISCUSSION TITLE
Wrong date / time inside docker container

### DISCUSSION THREAD

**Discussion by hannes09** [coder/coder#9187] [General] [answered]
I use a docker container in my workspace.
Git repos are mounted as volumes in the container.
The date inside the container is wrong. 
Also my git commits have wrrong time.

I tried to get host time by:

`volumes:
    - "/etc/timezone:/etc/timezone:ro"
    - "/etc/localtime:/etc/localtime:ro"`

Still wrong time inside the container.
The host system has a correct time.

**Comment by matifali** [comment, marked as answer]
This could be a good resource to start with: https://hoa.ro/blog/2020-12-08-draft-docker-time-timezone
